utilities/pytest_matrix_utils.py

```
@@ -75,6 +75,15 @@ def immediate_matrix(matrix):
     return matrix_to_return
 
 
+def snapshot_import_cron_format_matrix(matrix):
+    matrix_to_return = []
+    for storage_class in matrix:
+        storage_class_name = [*storage_class][0]
```

Ruff (0.16.0)

[warning] Prefer `next(iter(storage_class))` over single element slice

Replace with `next(iter(storage_class))`

(RUF015)